azure-pipelines.yml

```diff
@@ -35,10 +39,11 @@ variables:
   # tests due to https://github.com/dotnet/spark/pull/711
   backwardCompatibleTestOptions_Windows_3: "--filter \
   (FullyQualifiedName!=Microsoft.Spark.E2ETest.IpcTests.DataFrameTests.TestDataFrameGroupedMapUdf)&\
-  (FullyQualifiedName!=Microsoft.Spark.E2ETest.IpcTests.DataFrameTests.TestGroupedMapUdf)"
+  (FullyQualifiedName!=Microsoft.Spark.E2ETest.IpcTests.DataFrameTests.TestGroupedMapUdf)&\
+  ($(backwardCompatiableTestIgnore_PR_810))"
   forwardCompatibleTestOptions_Windows_3: ""
   backwardCompatibleTestOptions_Linux_3: $(backwardCompatibleTestOptions_Windows_3)
-  forwardCompatibleTestOptions_Linux_3: $(forwardCompatibleTestOptions_Linux_2_4)
```

Comment:
So this is an existing bug? @suhsteve do you know anything about this why this flag was here?

Reply:
This is safe to remove.  Seems like the Test Attribute skips Spark 3.0 tests.  https://github.com/dotnet/spark/blob/b16b4f64e6978432f0c3b7976f7a6153300c7bd9/src/csharp/Extensions/Microsoft.Spark.Extensions.Hyperspace.E2ETest/HyperspaceTests.cs#L57-L58

Reply:
Cool, thanks.